tests/rust/test_repo.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 The MobileCoin Foundation
+from cookiecutter.main import cookiecutter
+from filecmp import dircmp
+
+
+def dirs_the_same(dirs):
+    """
+    Returns true when `dirs` and all its subdirectories are the same
+    """
+    if dirs.diff_files != []:
+        return False
+
+    return all(dirs_the_same(d) for d in dirs.subdirs.values())
+
+
+def test_default_repo_render(template_root, tmp_path):
+    cookiecutter(
+        str(template_root),
+        directory="rust/repo",
+        no_input=True,
+        default_config=True,
+        output_dir=tmp_path,
+    )
+
+    dirs = dircmp(tmp_path, template_root / "tests/rust/repo/expecteds/default")
```

Comment:
I used "default" instead of "basic"
I'm not too particular here, but my plan going forward is to compare most things against "default". i.e the GPL test would compare against "default" and ensure only the license file and cargo.toml file differ. then it would compare it's license file to an expected GPL license file, and probably parse it's cargo.toml and ensure that it's license property is for GPL